x/concentrated-liquidity/keeper_test.go

```diff
@@ -434,47 +432,6 @@ func (s *KeeperTestSuite) TestValidatePermissionlessPoolCreationEnabled() {
 	s.Require().Error(s.App.ConcentratedLiquidityKeeper.ValidatePermissionlessPoolCreationEnabled(s.Ctx))
 }
 
-// runFungifySetup Sets up a pool with `poolSpreadFactor`, prepares `numPositions` default positions on it (all identical), and sets
-// up the passed in incentive records such that they emit on the pool. It also sets the largest authorized uptime to be `fullChargeDuration`.
-//
-// Returns the pool, expected position ids and the total liquidity created on the pool.
-func (s *KeeperTestSuite) runFungifySetup(address sdk.AccAddress, numPositions int, fullChargeDuration time.Duration, poolSpreadFactor osmomath.Dec, incentiveRecords []types.IncentiveRecord) (types.ConcentratedPoolExtension, []uint64, osmomath.Dec) {
```

Comment:
let's keep this for now, I am planning to move these helper methods into app testing suite for future uses